src/controllers/user.controller.ts

```diff
@@ -97,7 +97,6 @@ export class UserController {
 
       return res.sendStatus(StatusCodes.BAD_REQUEST);
```

Comment:
Don't return BAD_REQUEST here. If the code arrives here then it's more like INTERNAL_SERVER_ERROR, because from what I see it's not supposed to arrive here (it's like unhandled scenario)